index.html

```diff
@@ -445,15 +445,17 @@ <h3>Claims</h3>
 
       <p>
 The data model for <a>claims</a> described above is powerful and can be used to
-express a large variety of statements. For example, whether or not someone is
-over the age of 21 may be expressed as follows:
+express a large variety of statements. For example, a claim specifying the data of birth of a subject
+can be used to prove that its age lies in an interval or is greater or less than a some value which is
+needed in use cases like proving drinking age or teenager discount or elderly discount;
+the holder has the flexibility to use the claim in any way that is applicable to that claim:
       </p>
 
       <figure>
         <img style="margin: auto; display: block;"
           width="50%" src="diagrams/claim-example.svg">
         <figcaption style="text-align: center;">
-An example of a basic claim that expresses that Pat is over the age of 21.
+An example of a basic claim that expresses that Pat's date of birth is 1534417926548 milliseconds from epoch.
```

Comment:
This should use an ISO8601 dateTime string like other examples.  Probably also just use schema.org "birthDate" property too. Also the new SVG is styled different than the other images and the subtext is wrapped oddly.

Reply:
The dateTime string is intentionally formatted since the exact encoiding of the value is defined in a schema.

Reply:
This is an introductory document.  I think it should be using easy to understand examples.  We should avoid an arbitrary UNIX timestamp that requires an explanation in both the caption and graphic.  Yes, that is a possible value, but it seems like something that could be in a more advanced example.  In this case using https://schema.org/birthDate with a readable ISO8601 date makes more sense.  The value could just be "1985-10-26" without the subtext and people will likely understand without introducing concepts like the epoch.

Reply:
Yeah, Google Docs. Specifically, Google Draw. Originals are here: https://drive.google.com/drive/u/1/folders/0BwQ7X2CScoA-emRfSmVfSU4xNTg

Reply:
@davidlehn Ok, i will update the birthDate to show ISO8601 format and clarify that format needs to be specified in the schema.